src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
-import App from 'src/App.jsx';
+import App from './App.jsx';
```

Comment:
Why changed to relative path?

Reply:
It was showing me error that's why i changed this.

Reply:
I see, but its not a good idea to use relative path here. Maybe try  add another '/' before the src? So make the path `/src/App.jsx` and see if that works